### PR TITLE
Automatically refit the service graph on resize.

### PR DESCRIPTION
**Pull request by cfcosta**
Automatically refits the cytoscape graph on resize, trying to keep everything on screen.

Bad video (too fast, but it should show it changing automatically when the window size changes.

### CODE REVIEW

File: src/components/CytoscapeLayout/CytoscapeLayout.tsx [hunk 1]
```diff
@@ -50,10 +52,15 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
...
   cyRef(cy: any) {
     this.cy = cy;
+
     cy.on('tap', 'node', (evt: any) => {
       let node = evt.target;
       console.log('clicked on: ' + node.id());
     });
+
+    cy.on('resize', (evt: any) => {
+            this.cy.fit();
```
Comment (mtho11) [inline src/components/CytoscapeLayout/CytoscapeLayout.tsx]: spacing is off here. Prettier says confirms this.

File: src/components/CytoscapeLayout/CytoscapeLayout.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ type CytoscapeLayoutProps = {
 
 export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProps, CytoscapeLayoutState> {
   cy: any;
+  dimensions: [number, number];
```
Comment (mtho11) [inline src/components/CytoscapeLayout/CytoscapeLayout.tsx]: dimensions is unused as it is declared later in a function with `let`

**Comment by josejulio** [comment]
> trying to keep everything on screen

Not sure if this PR does that, but I think that as an user i would be annoyed if i lose my zoom level when resizing the window.

**Comment by mwringe** [comment]
The service graph resize seems to be working for me. Do we still need this?

**Comment by mwringe** [comment]
Trying out this PR, I don't think it works really as expected, there are some weird behaviours if you are zoomed in and go to resize this screen.
This needs more investigation.

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

**Comment by israel-hdez** [comment]
Closing, because the graph sizing seems to be already fixed.

**Pull request closed without merging**